main.py

```diff
@@ -344,53 +374,95 @@ def __init__(self, *args):
 		self.inst = Xqemu()
 		self.settings = SettingsManager()
 		self.settings.load()
-		self.runButton.setText('Start')
-		self.pauseButton.setText('Pause')
-		self.screenshotButton.setText('Screenshot')
+		self.fileformats = [
+			{
+				'name': 'ISO9660 CD/DVD image file',
+				'offset': [
+					0x8001,
+					0x8801,
+					0x9001
+				],
+				'signature': [
+					'43 44 30 30 31'
+				]
+			},
+			{
+				'name': 'XISO',
+				'offset': [
+					0x10000
+				],
+				'signature': [
+					" ".join("{:02x}".format(ord(c)) for c in 'MICROSOFT*XBOX*MEDIA')
```

Comment:
Same comment as above

Reply:
same as above

Reply:
This is a unicode string. `'MICROSOFT*XBOX*MEDIA'.encode('utf-16')` or something will do the trick.
(Microsoft / Xbox used UCS-2 encoding normally)